### DISCUSSION TITLE
About add me to the team

### DISCUSSION THREAD

**Discussion by aleksiby** [ROM-builders/temporary#8578] [General]
Hi i was successfully build yesterday, here's the link 
https://cirrus-ci.com/build/6372782593277952

And here's my resource :
Tree : https://github.com/aleksiby/platform_device_xiaomi_joyeuse
Common : https://github.com/aleksiby/platform_device_xiaomi_sm6250-common

Please add me to the team sar, my telegram account is @nekoskuy

Thanks

**Comment by Apon77** [comment]
> Hi i was successfully build yesterday, here's the link https://cirrus-ci.com/build/6372782593277952
> 
> And here's my resource : Tree : https://github.com/aleksiby/platform_device_xiaomi_joyeuse Common : https://github.com/aleksiby/platform_device_xiaomi_sm6250-common
> 
> Please add me to the team sar, my telegram account is @nekoskuy
> 
> Thanks

https://t.me/ROM_builders/72224
